integration_test/ops_agent_test.go

```diff
@@ -1515,50 +1594,46 @@ func TestTCPLog(t *testing.T) {
 		if err := agents.SetupOpsAgent(ctx, logger.ToMainLog(), vm, config); err != nil {
 			t.Fatal(err)
 		}
-		command := `echo '{"msg":"test tcp log 1"}{"msg":"test tcp log 2"}' | /opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit -i stdin -o tcp://127.0.0.1:5170 -p format=json_lines && echo '{"msg":"test tcp log 3"}{"msg":"test tcp log 4"}' | /opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit -i stdin -o tcp://127.0.0.1:5170 -p format=json_lines`
-		// Installing Netcat or equivalent tools on Windows is tricky so we can instead
-		// use this Powershell Script that can send TCP messages.
-		if gce.IsWindows(platform) {
-			command = `$ErrorActionPreference = 'Stop'
-			function Send-TcpString {
-			  param(
-			    [string]$TargetIP,
-			    [int]$TargetPort,
-			    [string]$Message
-			  )
-			  $Socket = New-Object System.Net.Sockets.TCPClient($TargetIP, $TargetPort) 
-			  $Stream = $Socket.GetStream() 
-			  $Writer = New-Object System.IO.StreamWriter($Stream)
-			  $Message | % {
-			    $Writer.WriteLine($_)
-			    $Writer.Flush()
-			  }
-			  $Stream.Close()
-			  $Socket.Close()
-			}
-			Send-TcpString 127.0.0.1 5170 '{"msg":"test tcp log 1"}{"msg":"test tcp log 2"}'
-			Send-TcpString 127.0.0.1 5170 '{"msg":"test tcp log 3"}{"msg":"test tcp log 4"}'`
+
+		// Start a background fluent-bit that outputs to TCP.
+		// The TCP receiver in the Ops Agent already parses to JSON,
+		// so don't double-parse it in the background fluent-bit.
+		var pipePath string
```

Comment:
It's simpler to just move `startFluentBitBackgroundPipe` outside the if statement:
```
pipePath, err := startFluentBitBackgroundPipe
if err != nil {
```

Reply:
Oops, good catch! Fixed